cli/src/main/kotlin/StartCommand.kt

```diff
@@ -45,41 +45,34 @@ class StartCommand(private val config: OrtServerOptions) : SuspendingCliktComman
         help = "The ID of the repository."
     ).long().required()
 
-    private val revision by option(
-        "--vcs-revision",
-        envvar = "VCS_REVISION",
-        help = "Revision of the repository to start the run for."
-    ).required()
-
-    private val path by option(
-        "--vcs-sub-path",
-        envvar = "VCS_SUB_PATH",
-        help = "Optional VCS sub-path of the repository."
-    )
-
-    private val jobConfigContext by option(
-        "--job-config-context",
-        envvar = "JOB_CONFIG_CONTEXT",
-        help = "Configuration context to use for the run."
-    )
-
     private val wait by option(
         "--wait",
         envvar = "WAIT",
         help = "Wait for the run to finish."
     ).flag()
 
+    private val ortRunConfigFile by option(
+        "--ort-run-config-file",
```

Comment:
This command will be called like
```
osc run start --ort-run-config-file ...
```

I'm not sure about the option naming, because you're somehow duplicating the `run` part, and I'm not sure if this is really for setting run __configurations__.

How about `--parameters-file`? Or is this to generic?